packages/jest-diff/src/index.js

```diff
@@ -30,6 +31,7 @@ const PLUGINS = [
   ReactTestComponentPlugin,
   ReactElementPlugin,
   AsymmetricMatcherPlugin,
+  ...ImmutablePlugins,
```

Comment:
This will fail on Node 4, as array spread is supported only behind a flag. You can use `concat`.